lib/transform.js

```diff
@@ -64,7 +64,7 @@ TreeTransformer.prototype = new TreeWalker;
                     x = this;
                     descend(x, tw);
                 } else {
-                    tw.stack[tw.stack.length - 1] = x = this.clone();
+                    tw.stack[tw.stack.length - 1] = x = this;
                     descend(x, tw);
```

Comment:
I removed it because the code doesn't work with clone. No one was using the after callback; it seemed the right thing to do.

I'd be curious If you could make it work and pass all tests with clone.